Sources/MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -189,7 +189,7 @@ public class CarPlayMapViewController: UIViewController {
     func setupNavigationMapView() {
         let navigationMapView = NavigationMapView(frame: UIScreen.main.bounds, navigationCameraType: .carPlay)
         navigationMapView.delegate = self
-        navigationMapView.mapView.mapboxMap.onNext(.styleLoaded) { _ in
```

Comment:
I think `CarPlayNavigationViewController` contains similar logic in `setupNavigationMapView`, which should be updated.

Reply:
Thanks, I fixed it to match what changed here.